dandi/tests/test_upload.py

```diff
@@ -212,6 +212,20 @@ def test_upload_bids_validation_ignore(
     dandiset.get_asset_by_path("sub-Sub1/anat/sub-Sub1_T1w.nii.gz")
 
 
+def test_upload_bids_metadata(
+    mocker: MockerFixture, bids_dandiset: SampleDandiset
+) -> None:
+    bids_dandiset.upload(existing="force")
```

Comment:
why upload needs to be forced?

Reply:
oh, you always do that... not sure if that is a good/clean practice, not liking it.. but not unique to this test, so let's proceed.